template/CordovaApp.Phone.jsproj

```diff
@@ -79,7 +79,9 @@
     <AppxManifest Include="package.phone.appxmanifest">
       <SubType>Designer</SubType>
     </AppxManifest>
-    <Content Include="images\*.png" Exclude="images\*.scale-180.*" />
+    <Content Exclude="images\*SplashScreen.*" Include="images\*.png" />
```

Comment:
We are excluding images named like `SplashScreen.jpg` or `SplashScreen.scale-100.png` here for phone project (we can do this taking into account that phone and desktop projects have different hardcoded splashscreen names).